database/seeds/CourseSeeder.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Course;
+use Illuminate\Database\Seeder;
+
+class CourseSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        factory(Course::class, 500)->create()->each(function (Course $course): void {
+            $course->save(factory(Course::class)->make()->toArray());
```

Comment:
To be honest, I don't understand what's happening here. Could you explain me why are you putting array of course's attributes into saving method of the course?

Reply:
I was under impression I need to do so to overwrite mass-assignment protection (e.g. for fetching creating_at attribute), but it seems it's actually off by default in Seeder classes.
I'll just refactor it to $model->save();

Reply:
I think you helped yourself with mass assignment problem when you've added `$fillable` array to the `Course` model. ;)

Reply:
I just added all the fields to $guarded array instead and I can still seed the database, so it seems like Seeders just don't care about mass assignment at all.